### DISCUSSION TITLE
Best practices

### DISCUSSION THREAD

**Discussion by MMAcode** [freeplane/freeplane#1138] [Show and tell]
I think it could be great to have a place where we can share our best practices with the Freeplane, especially if these may not be obvious ones. 

I will start by sharing mine here. (I am using Freeplane for over 10-15 years now!) I felt I truly utilised its power when I was in university writing my Master's thesis. I think I had a great system for collecting scientific notes, so I will describe it here. 

### How I managed for my Master thesis to be directly proposed to PhD dissertation thanks to Freeplane...

1) When I read a specific scientific article (or book), I kept **all notes/nodes for a specific article/book in one single branch**. With every note I also wrote down the **page** from which the text was (in node details).
2) After I finished reading the article/book, I selected all the notes from that book (all nested sub-branch of the main branch) and added **attribute to all of those nodes with the book name and author**. Like this, each individual node/note from the article was carrying information about its origin (page and article).
3) Once labelled as described above, I was able to move any notes anywhere. I started **grouping notes according to their topics**, regardless of which source those were from. Like this, I was able to have for example 50 notes from 10 different resources on a single topic (eg "gratitude journal").
4) Once I have all notes organised in these topics (and subtopics, and sub-subtopics...) and those topics made sense to me, I finally started **writing** my thesis. Like this, it was super easy to write on one topic while **referencing** all the resources exactly (thanks to author's and article's name in the attribute) and also including pages those notes came from (thanks to node details).
-> Thanks to this approach I was able to write well-organised, well-referenced text.

(Btw, there is a Freeplane-based software called [SciPlore](https://sourceforge.net/projects/splmm/) which was designed for precisely this purpose, but I never bought into that one.)

**Comment by dpolivaev** [comment]
Thanks for sharing your success story. By the way, would you like to share any of your maps?

**Comment by gravmath** [comment]
I am intrigued by your approach but I have some basic questions.  Let's say that you were researching 'running' and your first source was XYZ.  Would you have a running mind map with an XYZ branch within it?  If so, then it seems you would also have another branch from the ABC reference also within it the running mind map; is this correct?  And suppose while reading XYZ, you came across what you judged to be an important point on something outside your topic and children subtopics (e.g., a small paragraph on the importance of new laws to protect runners from drunk drivers), how would you track that?  Would it go into a different mind map?  Note that I am fairly new to Freeplane so answer (and judge) accordingly.  Thanks in advance.

**Reply by MMAcode** [reply]
Hi @gravmath, sorry, I am not sure I understand. Check the images below and if that does not clarify anything, feel welcome to reiterate your question.
I have put together a few visuals for illustration (as requested by @dpolivaev).
1)
![image](https://github.com/freeplane/freeplane/assets/52301171/8c61faa0-7b91-47ed-b3ed-b2b223d6ac33)
2)
![image](https://github.com/freeplane/freeplane/assets/52301171/b274f1f3-91dd-4c69-8b25-ffac617eacb8)
3)
![image](https://github.com/freeplane/freeplane/assets/52301171/d86e3f7e-4670-4309-8b18-701602199bba)
4)
![image](https://github.com/freeplane/freeplane/assets/52301171/62cbc4bf-6ac6-4051-86fe-72cfcf2d390f)
5)
![image](https://github.com/freeplane/freeplane/assets/52301171/46e028d1-2e24-4acb-ab7f-96260cb94dab)
(Once a note is in the 'sorted' section, it can be moved or copied anywhere within that section.

I hope this helps, at least a bit. :-)

**Reply by gravmath** [reply]
Hi @MMAcode.  I want to thank for that very thorough reply.  I will start experimenting with your approach and may write something up for this best practices thread based on results.  

My primary use case is for mathematical models.  Different authors have quite different observations about the same things and there are often far different expressions.  

To be more concrete, in the field of orbital mechanics, there are many different ways of expressing orbital motion and the perturbations to it.  For example, the (specific) energy can be written in terms of ordinary position and velocity, or in terms of Keplerian elements, or etc.  

In a similar way, the equations of fluid mechanics, while describing the same physical effects look different from a Eulerian v. Lagrangian point-of-view and whether the equations are written in flux-conservative or non-flux-conservative form.  

I am trying to knowledge manage when I teach and mentor and I often forget critical linkages.

**Comment by kenfhill84083** [comment]
> I think it could be great to have a place where we can share our best practices with the Freeplane, especially if these may not be obvious ones.
> 
> I will start by sharing mine here. (I am using Freeplane for over 10-15 years now!) I felt I truly utilised its power when I was in university writing my Master's thesis. 

Thanks.  This encouraged me to start a mindmap of Freeplane best practices with yours as the 1st entry in a branch called "use cases".  I packed your full text into the notes of your branch then used keywords that made sense to me.  I'm never going to write a thesis but I like your approach.

BTW : I am a big believer in gratitude journals - I keep mine in Freeplane.

[Freeplane best practices.zip](https://github.com/freeplane/freeplane/files/11635443/Freeplane.best.practices.zip)
